scripts/automation/upload_manual.py

```
@@ -0,0 +1,233 @@
+    if token:
+        return token
+    return getpass.getpass("アップロードトークンを貼り付けてEnter（表示されません）: ").strip()
+
+
+def upload(base_url: str, manual_id: str, html_path: str, token: str) -> dict:
+    """PUT /manuals/:id を実行し、レスポンスのJSONを返す。"""
+    with open(html_path, "rb") as f:
+        body = f.read()
+
+    url = f"{base_url.rstrip('/')}/manuals/{manual_id}"
+    req = urllib.request.Request(url, data=body, method="PUT")
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)